dali/test/python/test_fw_iterators.py

```diff
@@ -344,7 +345,8 @@ def check_mxnet_iterator_pass_reader_name(shards_num, pipes_number, batch_size,
 
     if batch_size > data_set_size // shards_num and last_batch_policy == LastBatchPolicy.DROP:
         assert_raises(AssertionError, MXNetIterator, pipes, [
-                      ("ids", MXNetIterator.DATA_TAG)], reader_name="Reader", last_batch_policy=last_batch_policy)
+                      ("ids", MXNetIterator.DATA_TAG)], reader_name="Reader", last_batch_policy=last_batch_policy,
+                      glob="It seems that there is no data in the pipeline. This may happen if `last_batch_policy` is set to PARTIAL and the requested batch size is greater than the shard size.")
```

Comment:
I don't think we should put verbatim copies of the entire message here - just enough to make sure it's the error we expect - like:
```suggestion
                      glob="It seems that there is no data in the pipeline*last_batch_policy*")
```

Reply:
Fixed